code/_core/obj/item/spellswap/_spellswap.dm

```diff
@@ -28,6 +28,20 @@
 		if(desired_choice != "Yes")
 			caller.to_chat(span("notice","You decide not to spellswap \the [I.name]."))
 			return TRUE
+		if(desired_projectile==null)
+			if(I.stored_spellswap)
+				qdel(I.stored_spellswap)
+				src.drop_item(I)
+				I.stored_spellswap = src
```

Comment:
``I.stored_spellswap`` = src should go before ``src.drop_item(I)``

Reply:
Fixed